lib/teleterm/api/proto/buf.yaml

```diff
@@ -4,6 +4,10 @@ lint:
     - DEFAULT
   except:
     - RPC_RESPONSE_STANDARD_NAME
+    - RPC_REQUEST_RESPONSE_UNIQUE
+    # TODO(ravicious): Fix protos and re-enable linters below.
+    - PACKAGE_DIRECTORY_MATCH
+    - RPC_REQUEST_STANDARD_NAME
```

Comment:
Thanks, I'll create a ticket for that tomorrow.

In the webapps repo we also have some additional proto files but they're completely orthogonal to the Go codebase. Still, it'll be worthwhile to use the same process for them as for any other protos. I'll create an issue for that too, but this will have to wait until the webapps repo is moved back into the teleport repo.

Reply:
Sounds good, thanks!